parsers/cxx/cxx_parser_template.c

```diff
@@ -269,7 +269,10 @@ cxxParserParseTemplateAngleBracketsInternal(bool bCaptureTypeParameters,int iNes
 	// around but without proper state (include headers, macro expansion, full type
 	// database etc) we simply can't do the same. However, we try to recover if we
 	// figure out we (or the programmer?) screwed up.
-	//
+    // For 'greater-than' operators, the first non-nested operator is taken as the
```

Comment:
A minor request.
Could you use "tab" characters instead of space characters for indentation?

Reply:
✅